### ISSUE TITLE
CSS and JS not found

### ISSUE THREAD

**Issue by hanisirfan** [opensourcepos/opensourcepos#3175]
### Background information

__IMPORTANT: If you choose to ignore this issue report template, your issue will be closed as we cannot help without the requested information.__

Please make sure you tick (add an x between the square brackets with no spaces) the following check boxes:

- [x] Reporting an issue of an unmodified OSPOS installation
- [x] Checked [open and closed issues](https://github.com/opensourcepos/opensourcepos/issues?utf8=%E2%9C%93&q=is%3Aissue) and no similar issue was already reported (please make sure you searched!)
- [x] Read [README](https://github.com/opensourcepos/opensourcepos/blob/master/README.md), [WHATS_NEW](https://github.com/opensourcepos/opensourcepos/blob/master/WHATS_NEW.txt), [INSTALL.md](https://github.com/opensourcepos/opensourcepos/blob/master/INSTALL.md) and [UPGRADE](https://github.com/opensourcepos/opensourcepos/blob/master/UPGRADE.txt)
- [x] Read the [FAQ](https://github.com/opensourcepos/opensourcepos#faq) for any known install and/or upgrade gotchas (in specific PHP extensions installed)
- [] Read the [wiki](https://github.com/opensourcepos/opensourcepos/wiki)
- [x] Executed any database upgrade scripts if an upgrade pre 3.0.0 (e.g. database/2.4_to_3.0.sql)
- [x] Aware the installation code is in [bintray](https://bintray.com/jekkos/opensourcepos/opensourcepos/view/files?sort=updated&order=asc#files) (see README), and [GitHub master](https://github.com/opensourcepos/opensourcepos/tree/master) is for [developers only](https://github.com/opensourcepos/opensourcepos/wiki/Development-setup) and therefore not complete nor stable

### Installation information

- OSPOS version is: 3.3.3
- OSPOS git commit hash is:  080e8ac20c2aa2b4677f996b13eacb6f71bdffa0
- PHP version is: 7.4
- MySQL or MariaDB version is: -
- OS and version is: -
- WebServer is: Apache
- Selected language is: -
- (If applicable) Docker installation: -
- (If applicable) Installation package for the LAMP/LEMP stack is: -

### Issue / Bug  / Question / New Feature

Please write your issue here. If a bug, please make sure to provide as much information as possible including configuration settings (e.g. Decimals set, Tax mode), language and steps to reproduce the bug.

I'm installing this on InfinityFree hosting. I only uploaded the files to it's default `htdocs` folder. The login page looks fine to me and when I'm logged in, the dashboard looks horrendous.

![image](https://user-images.githubusercontent.com/66242389/116374052-881e6a00-a840-11eb-9225-5d8a8b0d1bff.png)

I open up the console and it looks like the CSS and JS file could't be found.

![image](https://user-images.githubusercontent.com/66242389/116373102-a3d54080-a83f-11eb-825c-10c037e99b7a.png)

Also, how can I use this on the root URL not `domain.com/public`. I've read the FAQ but I couldn't understand it since I'm not knowledgeable in .htaccess config.

**Comment by jekkos** [comment]
Which download did you use? The one from the releases section? Can you check if the files are present at all in your install?

**Comment by hanisirfan** [comment]
Yes, I've downloaded one from the release section. The release version `3.3.4`. I believe I download the ZIP file.

![image](https://user-images.githubusercontent.com/66242389/116377694-06304000-a844-11eb-951b-c720b7c3e02e.png)

For some reason, the folder I download name `opensourcepos-3.3.4` but when I logged on, the footer shows `3.3.3 - dev`.
![image](https://user-images.githubusercontent.com/66242389/116377994-4b547200-a844-11eb-9d80-cc7b9454a4ff.png)

May I know where did the CSS and JS file located? From the get request shows it's on `dist` folder. Where is it? I seems to not find it. I only have these CSS files in `/htdocs/public/css/` folder.

![image](https://user-images.githubusercontent.com/66242389/116378440-b8680780-a844-11eb-923d-fa1774326b63.png)

**Comment by hanisirfan** [comment]
Downloaded the same ZIP file and I can't seems to find the files from the console error.
![image](https://user-images.githubusercontent.com/66242389/116378744-fc5b0c80-a844-11eb-8013-07a304287b87.png)

**Comment by jekkos** [comment]
You have to download from the releases section, don't download the sources zip file (as described in the README.md). It does not contain any minified files.

**Comment by jekkos** [comment]
Download this one instead: 
opensourcepos.20210422073326.master.392d92.zip
